Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -26,12 +26,12 @@ namespace o2 { namespace ITSMFT { class Point; }}  // lines 22-22
 namespace o2 { namespace ITS { class GeometryHandler; }}
 namespace o2 { namespace ITS { class MisalignmentParameter; }}
 namespace o2 { namespace ITS { class GeometryTGeo; }}
-namespace o2 { namespace ITS { class V1Layer; }}  // lines 23-23
+namespace o2 { namespace ITS { class V3Layer; }}  // lines 23-23
```

Comment:
I think at @matthiasrichter is referring to the comments `// lines 23-23`. Indeed they don't carry much information and the line numbers are likely to be out of sync. So I would just remove them.